### PR TITLE
vstart.sh: unify the indent

### PR DESCRIPTION
**Pull request by richael02**
Change-Id: Ie1dbe25f81609e5b2f6751da9e87e191a2f78bc2
Signed-off-by: Richael Zhuang <richael.zhuang@arm.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by sebastian-philipp** [comment]
Downside of this PR would be that `git blame` on HEAD would no longer show the original author. 

is it worth it?

**Comment by tchaikov** [comment]
retest this please.

File: src/vstart.sh [hunk 1]
```diff
@@ -542,69 +544,71 @@ ms bind msgr1 = true
...
-                fi
-                if [ $(get_pci_selector_num) -lt $CEPH_NUM_OSD ]; then
-                    echo "OSD number ($CEPH_NUM_OSD) is greater than NVME SSD number ($(get_pci_selector_num)), please check." >&2
-                    exit
-                fi
-                BLUESTORE_OPTS="        bluestore_block_db_path = \"\"
+    fi
+    if [ "$short" -eq 1 ]; then
+        COSDSHORT="
+        osd max object name len = 460
```
Comment (tchaikov) [inline src/vstart.sh]: this changes the behavior of this script. and it's wrong.

Comment (richael02) [reply src/vstart.sh]: I thought that adding a new line above it was OK. Well  I'll modify it to keep the original one.

File: src/vstart.sh [hunk 2]
```diff
@@ -776,36 +776,36 @@ EOF
...
             local key_fn=$CEPH_DEV_DIR/osd$osd/keyring
-	    cat > $key_fn<<EOF
+            cat > $key_fn<<EOF
 [osd.$osd]
-	key = $OSD_SECRET
+        key = $OSD_SECRET
 EOF
             echo adding osd$osd key to auth repository
-            ceph_adm -i "$key_fn" auth add osd.$osd osd "allow *" mon "allow profile osd" mgr "allow profile osd"
+                ceph_adm -i "$key_fn" auth add osd.$osd osd "allow *" mon "allow profile osd" mgr "allow profile osd"
```
Comment (tchaikov) [inline src/vstart.sh]: wrong indent.

File: src/vstart.sh [hunk 3]
```diff
@@ -890,39 +890,39 @@ start_mds() {
...
+            fi
+            prun $SUDO "$CEPH_BIN/ceph-authtool" --create-keyring --gen-key --name="mds.$name" "$key_fn"
+            ceph_adm -i "$key_fn" auth add "mds.$name" mon 'allow profile mds' osd 'allow rw tag cephfs *=*' mds 'allow' mgr 'allow profile mds'
+            if [ "$standby" -eq 1 ]; then
+                prun $SUDO "$CEPH_BIN/ceph-authtool" --create-keyring --gen-key --name="mds.${name}s" \
+                     "$CEPH_DEV_DIR/mds.${name}s/keyring"
+                ceph_adm -i "$CEPH_DEV_DIR/mds.${name}s/keyring" auth add "mds.${name}s" \
+                             mon 'allow profile mds' osd 'allow *' mds 'allow' mgr 'allow profile mds'
 	    fi
+	fi
```
Comment (tchaikov) [inline src/vstart.sh]: please avoid use tab in this file.

**Comment by tchaikov** [comment]
retest this please.

**Comment by tchaikov** [comment]
- http://pulpito.ceph.com/kchai-2019-06-02_15:00:54-rados-wip-kefu-testing-2019-06-01-2346-distro-basic-mira/
- http://pulpito.ceph.com/nojha-2019-06-04_22:13:54-rados-wip-kefu-testing-2019-06-01-2346-distro-basic-smithi/

the valgrind failure is a known issue tracked by http://tracker.ceph.com/issues/38403, and it should be addressed by #28397

**Comment by tchaikov** [comment]
@richael02 i rebased your change onto master and added two commits, could you please take a look?

**Comment by richael02** [comment]
@tchaikov It looks good to me.

**Pull request merged**